bundles/org.openhab.binding.bmwconnecteddrive/src/main/java/org/openhab/binding/bmwconnecteddrive/internal/handler/RemoteServiceHandler.java

```diff
@@ -76,7 +84,8 @@
         HORN_BLOW(REMOTE_SERVICE_HORN, "Horn Blow"),
         CLIMATE_NOW(REMOTE_SERVICE_AIR_CONDITIONING, "Climate Control"),
         CHARGE_NOW(REMOTE_SERVICE_CHARGE_NOW, "Start Charging"),
-        CHARGING_CONTROL(REMOTE_SERVICE_CHARGING_CONTROL, "Send Charging Profile");
+        CHARGING_CONTROL(REMOTE_SERVICE_CHARGING_CONTROL, "Send Charging Profile"),
+        CHARGING_PROFILE("chargingProfile", "Send Charging Profile");
```

Comment:
It seems this the only one that received a string literal instead of constant as a first parameter.

Reply:
Ouch, leftover from my several tryings to send charge profile. Cleaned up.